index.js

```diff
@@ -128,3 +133,18 @@ function byLocation(location) {
 function getCoercer(type) {
   return COERCION_STRATEGIES[type];
 }
+
+function pathsep(format) {
+  switch (format) {
+    case 'csv':
```

Comment:
Hmm.  I don't think these are valid item types for arrays.  See https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json#L1092.

Is it possible to handle this with middleware outside this project?